site/docs/supported-models/_components/llm-models-table/models.ts

```diff
@@ -115,13 +115,12 @@ export const LLM_MODELS: LLMModelType[] = [
           'https://huggingface.co/CohereForAI/aya-23-8B',
           'https://huggingface.co/CohereForAI/aya-expanse-8b',
           'https://huggingface.co/CohereForAI/aya-23-35B',
-          'https://huggingface.co/CohereForAI/aya-expanse-35b',
         ],
       },
       {
         name: 'C4AI Command R',
         links: [
-          'https://huggingface.co/CohereForAI/c4ai-command-r7b-12-2024',
+          'https://huggingface.co/CohereLabs/c4ai-command-r7b-12-2024',
           'https://huggingface.co/CohereForAI/c4ai-command-r-v01',
```

Comment:
Should we change organization from `CohereForAI` to `CohereLabs` for all other Cohere models?

Reply:
Changed